packages/web-react/README.md

```diff
@@ -69,6 +69,16 @@ are doing so at your own risk.
 
 Please consult additional styling with [web package documentation][web-pkg-rebrand].
 
+## Development
```

Comment:
Last modification before approve. Please move the development and section to the new `CONTRIBUTING.md` in every package.

Usage is as follows:
- README - main information for users, eg. for those using this package
- CONTRIBUTING - main information for developers, eg. for those developing this package

Thanks.

Reply:
Development and Testing, both to contributing, please.